### PR TITLE
seekBar with tests max value is undefined

### PR DESCRIPTION
**Pull request by AlinaLysenko**

### CODE REVIEW

**Comment by mf-code** [comment]
[6233] - hTzXDF6UYTcG

**Comment by mf-code** [comment]
[6233] - qccAbT7Rb2CU

**Comment by mf-code** [comment]
[6233] - 6YsSavAuzx3b

**Comment by mf-code** [comment]
[6233] - Bo2cpGnBhhno

**Comment by mf-code** [comment]
[6233] - kp9gVstIP9MI

**Comment by mf-code** [comment]
[6233] - TkYKznmdWnEj

**Comment by mf-code** [comment]
[6233] - PSAPNSAGucQ7

**Comment by mf-code** [comment]
[6233] - CHV5KuOSW8ub

**Comment by mf-code** [comment]
[6233] - QaRoSECogW2i

**Comment by mf-code** [comment]
[6233] - 6dmxkJnAhMqD

**Comment by mf-code** [comment]
[6233] - L4ZjhsWlB9rV

**Comment by mf-code** [comment]
[6233] - QTZXhimUqfzC

**Comment by mf-code** [comment]
[6233] - F47tFEcntW95

**Comment by mf-code** [comment]
[6233] - jETGyrjnnZJF

**Comment by mf-code** [comment]
[6233] - vXrZaSKEpXuS

**Comment by mf-code** [comment]
[6233] - Z65ElNbgIFCe

**Comment by mf-code** [comment]
[6233] - oExzciHptZAP

**Comment by mf-code** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/app/android/SeekBar.java  1
- jdi-light-mobile-tests/src/test/java/nativeapp_android/tests/ToggleButtonTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6292802&bid=20471527)

File: jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/app/android/SeekBar.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    @JDIAction("Set the minimum value for '{name}'")
+    public void setMinimumValue() {
+        setSliderValue("0");
+    }
+
+    @JDIAction("Set the maximum value for '{name}'")
+    public void setMaximumValue() {
+        executeDriverMethod(PerformsTouchActions.class, (PerformsTouchActions driver) -> {
+            TouchAction<?> ta1 = new TouchAction<>(driver)
```
Comment (mf-code) [inline jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/app/android/SeekBar.java:27]: Codacy found an issue: [Avoid unused local variables such as 'ta1'.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6292802)

**Review by alexarm — APPROVED** [review]

**Comment by mf-code** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light-mobile-tests/src/test/java/nativeapp_android/tests/ToggleButtonTests.java  2
- jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/app/android/SeekBar.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6292802&bid=20471527)

File: jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/app/android/SeekBar.java (hunk 1 above)
Comment (mf-code) [inline jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/common/app/android/SeekBar.java:27]: Codacy found an issue: [Avoid unused local variables such as 'ta1'.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6292802)

**Pull request merged**